### PR TITLE
Intellij performance benchmark

### PR DESCRIPTION
**Pull request by dima-avdeev-jb**
Made performance tests with Compose tool windows panels in Idea

## CPU time in Activity Monitor.
0 Panels: 33
1 Panels: 37
2 Panels: 41
3 Panels: 44
4 Panels: 48
5 Panels: 52
6 Panels: 55

## Measurements results inside JVM:

0 panels:
memory: 181.7 MB
system load: 6.759 % 

1 panels:
memory: 184.2 MB
system load: 6.427 % 

2 panels:
memory: 184.6 MB
system load: 6.464 % 

3 panels:
memory: 184.6 MB
system load: 6.456 % 

4 panels:
memory: 184.6 MB
system load: 6.421 % 

5 panels:
memory: 184.8 MB
system load: 6.295 % 

6 panels:
memory: 185.1 MB
system load: 6.185 %

### CODE REVIEW

File: benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package com.jetbrains.compose.color
```
Comment (igordmn) [inline benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt:6]: Can we delete this file and HSVTest.kt? I suppose this is from another project.

Comment (igordmn) [inline benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt:6]: Can we delete this file and HSVTest.kt? I suppose this is from another project.

**Review by igordmn — APPROVED** [review]

File: benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt (hunk 1 above)
Comment (dima-avdeev-jb) [reply benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt:6]: Yes, sure!

Comment (dima-avdeev-jb) [reply benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt]: Done

Comment (dima-avdeev-jb) [reply benchmarks-intellij/src/test/kotlin/com/jetbrains/compose/color/ColorPickerUITest.kt]: Done

**Pull request merged**